### PR TITLE
fix(time-select): computed time-select value

### PR DESCRIPTION
**Pull request by siam-ese**
time-select组件modevalue更改时,里面的value没有同步更改

Please make sure these boxes are checked before submitting your PR, thank you!
[codepen bug demo](https://codepen.io/siamzz/pen/MWbOMgg)

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by conan2008** [comment]
changed computed ， but it still doesn't work

File: packages/time-select/src/time-select.vue [hunk 1]
```diff
@@ -144,7 +144,7 @@ export default defineComponent({
   emits: ['change', 'blur', 'focus', 'update:modelValue'],
   setup(props) {
     // data
-    const value = ref(props.modelValue)
+    const value = computed(() => props.modelValue)
```
Comment (BeADre) [inline packages/time-select/src/time-select.vue:147]: I guess the `value` is unnecessary, we can do `:model-value="modelValue"` in el-select

Comment (JeremyWuuuuu) [reply packages/time-select/src/time-select.vue:147]: Yep, that's the essential idea of not introducing more variables. The less the better.

**Comment by JeremyWuuuuu** [comment]
Checkout #1725, the value is till being wrapped into `computed`, rebase the `origin/dev` branch then get do something with the `computed` statement. You can actually remove it, the computed is just some sort duplication.

**Pull request closed without merging**